apm-agent-core/src/main/java/co/elastic/apm/report/serialize/DslJsonSerializer.java

```diff
@@ -85,11 +85,13 @@
     private static final byte NEW_LINE = (byte) '\n';
     private final Collection<String> excludedStackFrames = Arrays.asList("java.lang.reflect", "com.sun", "sun.", "jdk.internal.");
     static final int MAX_VALUE_LENGTH = 1024;
+    static final int MAX_SQL_VALUE_LENGTH = 10000;
     private static final Logger logger = LoggerFactory.getLogger(DslJsonSerializer.class);
     private static final String[] DISALLOWED_IN_TAG_KEY = new String[]{".", "*", "\""};
     // visible for testing
     final JsonWriter jw;
     private final StringBuilder replaceBuilder = new StringBuilder(MAX_VALUE_LENGTH);
+    private final StringBuilder sqlReplaceBuilder = new StringBuilder(MAX_SQL_VALUE_LENGTH);
```

Comment:
only use one replaceBuilder for both, long and default string values

Reply:
Fine Felix. I shall update now.

Reply:
I have initialized the value with MAX_LONG_STRING_VALUE_LENGTH for replaceBuilder as well. I hope it is fine.

private final StringBuilder replaceBuilder = new StringBuilder(MAX_LONG_STRING_VALUE_LENGTH);